stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/extensions/internal/User.kt

```diff
@@ -7,8 +7,8 @@ import io.getstream.chat.android.livedata.ChatDomain
 import io.getstream.chat.android.ui.R
 
 internal fun User.isCurrentUser(): Boolean {
-    return if (ChatDomain.isInitialized) {
-        id == ChatDomain.instance().currentUser.id
+    return if (ChatDomain.isInitialized && ChatDomain.instance().user.value != null) {
+        id == ChatDomain.instance().user.value!!.id
```

Comment:
I'd prefer writing `ChatDomain.instance().user.value` into a standalone `val` and reading its value in if statement to avoid `!!`

Reply:
I can also use `?`... no need for the `!!` at all.